packages/react-storage/src/components/StorageBrowser/displayText/types.ts

```diff
@@ -67,6 +67,7 @@ export interface DefaultLocationDetailViewDisplayText
   tableColumnNameHeader: string;
   tableColumnSizeHeader: string;
   tableColumnTypeHeader: string;
+  getActionListLabel: (key: string | undefined) => string;
```

Comment:
For discoverability think adding "Item" will help consumers to better understand that this is used for resolving list item labels:
```
getActionListItemLabel
```